plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetPreferences.java

```diff
@@ -59,7 +59,7 @@ public final class ResultSetPreferences {
     public static final String RESULT_SET_SHOW_BOOLEAN_AS_CHECKBOX = "resultset.show.boolean.checkbox"; //$NON-NLS-1$
     public static final String RESULT_SET_SHOW_WHITESPACE_CHARACTERS = "resultset.show.whitespace.characters"; //$NON-NLS-1$
     public static final String RESULT_SET_CLICK_TOGGLE_BOOLEAN = "resultset.show.boolean.toggleOnClick"; //$NON-NLS-1$
-    public static final String RESULT_SET_CALC_COLUMN_WIDTH_BY_VALUES = "resultset.calc.columnWidthByValues"; //$NON-NLS-1$
+    public static final String RESULT_SET_CALC_COLUMN_WIDTH_METHOD = "resultset.calc.columnWidthMethod"; //$NON-NLS-1$
```

Comment:
No backward compatibility with the old preference

Reply:
But the whole meaning was changed

Reply:
Saved settings will be broken, if we rename this preference.

Reply:
And what should be done in this situation? help please

Reply:
1. If the new preference is set, use its value
2. If the new preference is not set, use the value from the old preference

And make sure the new preference's default value results in the same behavior as before.